src/common/logging.cpp

```diff
@@ -147,7 +188,12 @@ void otbrLog(otbrLogLevel aLevel, const char *aLogTag, const char *aFormat, ...)
         }
         else
         {
+#if OTBR_ENABLE_PLATFORM_ANDROID
+            __android_log_print(ConvertToAndroidLogPriority(aLevel), LOG_TAG, "%s%s: %s", sLevelString[aLevel],
+                                GetPrefix(aLogTag), buffer);
+#else
             syslog(static_cast<int>(aLevel), "%s%s: %s", sLevelString[aLevel], GetPrefix(aLogTag), buffer);
```

Comment:
Could you show an example demonstrating there's no essential change between the old output and the new output?

Reply:
There are no changes on the output formats of these two log methods. For example:

Linux:

```
11-21 11:31:37.301  3879  3879 I ot-daemon: [NOTE]-AGENT---: Running OTBR_AGENT/Unknown
11-21 11:31:37.301  3879  3879 I ot-daemon: [NOTE]-AGENT---: Thread version: 1.3.0
11-21 11:31:37.301  3879  3879 I ot-daemon: [NOTE]-AGENT---: Thread interface: thread-wpan
11-21 11:31:37.301  3879  3879 I ot-daemon: [NOTE]-AGENT---: Backbone interface is not specified
11-21 11:31:37.301  3879  3879 I ot-daemon: [NOTE]-AGENT---: Radio URL: threadnetwork_hal://binder?none
```

Android:

```
11-21 11:24:05.345  3916  3916 I ot-daemon: [NOTE]-AGENT---: Running OTBR_AGENT/Unknown
11-21 11:24:05.345  3916  3916 I ot-daemon: [NOTE]-AGENT---: Thread version: 1.3.0
11-21 11:24:05.345  3916  3916 I ot-daemon: [NOTE]-AGENT---: Thread interface: thread-wpan
11-21 11:24:05.345  3916  3916 I ot-daemon: [NOTE]-AGENT---: Backbone interface is not specified
11-21 11:24:05.345  3916  3916 I ot-daemon: [NOTE]-AGENT---: Radio URL: threadnetwork_hal://binder?none
```

Reply:
Thanks!

Reply:
Better clarify this in the PR description.

Reply:
Updated